joomla/joomla-cms
# [4.0] [a11y] com_finder :: add missing legend to fieldset

Pull Request for Issue # .

### Summary of Changes

After a11y testing the search form of component com_finder using WAVE it alerted me with the following message:
_Fieldset missing legend_

This PR changes a couple of things:
- add a `<legend>` to the `<fieldset>` of the search field section
- change `<div>` into `<fieldset>` of the advanced search section
- add a `<legend>` to the `<fieldset>` of the advanced search section
- add className `sr-only` to both fieldsets to keep is visually hidden.

### Testing Instructions

- Apply search... this will lead you to the search results on com_finder
- Inspect element on the search form and notice there is no `<legend>` in the `<fieldset>`
- Apply this PR
- Inspect element on the search form and notice the changes from above

### Actual result BEFORE applying this Pull Request
Screenshot taken from my dev tools
<img width="842" alt="Schermafbeelding 2020-08-31 om 17 33 58" src="https://user-images.githubusercontent.com/639822/91738186-66d18500-ebb0-11ea-95e0-cd63d9f03f14.png">


### Expected result AFTER applying this Pull Request
Screenshot taken from my dev tools
<img width="849" alt="Schermafbeelding 2020-08-31 om 17 34 15" src="https://user-images.githubusercontent.com/639822/91738201-6b963900-ebb0-11ea-902f-b693c75de018.png">


### Documentation Changes Required
